cmd/access/node_builder/access_node_builder.go

```diff
@@ -612,8 +617,7 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionDataRequester() *FlowAccessN
 				node.RootChainID,
 				builder.executionDataConfig.InitialBlockHeight,
 				highestAvailableHeight,
-				builder.apiRatelimits,
-				builder.apiBurstlimits,
+				builder.unsecureGrpcServer,
```

Comment:
previously, this could be configured on a separate port. Please keep that functionality, but add support for configuring it on the same port.